staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -136,7 +139,7 @@ func (pq waitForPriorityQueue) Peek() interface{} {
 // ShutDown gives a way to shut off this queue
 func (q *delayingType) ShutDown() {
 	q.Interface.ShutDown()
```

Comment:
are the other ShutDown/Stop calls made already reentrant themselves?

Reply:
in the underlying calls?  yes